jiant/preprocess.py

```diff
@@ -412,6 +412,29 @@ def parse_task_list_arg(task_list):
     return task_names
 
 
+def parse_cuda_list_arg(cuda_arg):
```

Comment:
`preprocess.py` seems like an odd place for this ? Why not in main ? Or in some `options.py` file (like how fairseq does it -- it's a bit unintuitive at first to look in another file for the option definitions, but I kind of like it)

Reply:
It's where the rest of the config parsing code lives. I'll move it over to options.py